### DISCUSSION TITLE
Backup folder is empty

### DISCUSSION THREAD

**Discussion by rohitsingal1** [inventree/InvenTree#5294] [General]
In my "config.yaml" the backup folder is configured to /opt/inventree/data/backup

![image](https://github.com/inventree/InvenTree/assets/128972942/68753956-1fad-4c5c-bf5c-b04d9413b1c9)

Also in "Background Tasks" under "Successful Tasks" InvenTree.tasks.run_backup is success.

![image](https://github.com/inventree/InvenTree/assets/128972942/2b55b100-7776-4f0c-9157-60999fc8d64d)
After this if I navigate to /opt/inventree/data/backup the folder is empty.

Will the backup of folder /opt/inventree/data be sufficient to restore the data in future ? If I do a manual backup with a cron job

**Comment by rohitsingal1** [comment]
Got it sorted. Thanks

**Reply by SchrodingersGat** [reply]
What was the issue?

**Comment by rohitsingal1** [comment]
The backup task was not generating the necessary files in the backup folder. After manually changing the value of "Repeats" to -1 it seems to be working.
